training/README.md

```diff
@@ -0,0 +1,160 @@
+## Train the Model with Your Own Data
+
+This document provides instructions to train the model on Watson Machine Learning, an offering of IBM Cloud. The instructions in this document assume that you already have an IBM Cloud account. If not, please create an [IBM Cloud](https://ibm.biz/Bdz2XM) account. 
+
+- [Prepare Data for Training](#prepare-data-for-training)
+- [Train the Model](#train-the-model)
+- [Rebuild the Model-Serving Microservice](#rebuild-the-model-serving-microservice)
+
+## Prepare Data for Training
+
+To prepare your data for training complete the steps listed in [data_preparation/README.md](data_preparation/README.md).
+
+## Train the Model
+
+In this document `$MODEL_REPO_HOME_DIR` refers to the cloned MAX model repository directory, e.g.
+`/users/hi_there/MAX-Text-Sentiment-Classifier`. 
+
+### Install Local Prerequisites
+
+Open a terminal window, change dir into `$MODEL_REPO_HOME_DIR/training` and install the Python prerequisites. (Model training requires Python 3.6 or above.)
+
+   ```
+   $ cd training/
+
+   $ pip install -r requirements.txt
+    ... 
+   ```
+
+### Customize Model Specific Parameters
+If you wish to change the network architecture or training hyper-parameters like `epochs` etc, change the corresponding arguments in `$MODEL_REPO_HOME_DIR/training/training_code/training-parameters.sh`.
+
+
+### Run the Setup Script
+
+The `wml_setup.py` script prepares your local environment and your IBM Cloud resources for model training.
+
+1. Locate the training configuration file. It is named `max-text-classifier-training-config.yaml`.
+
+   ```
+
+   $ ls *.yaml
+     max-text-classifier-training-config.yaml 
+   ```
+
+1. Configure your environment for model training. Run `wml_setup.py` and follow the prompts.
+
+   ```
+    $ python wml_setup.py max-text-classifier-training-config.yaml 
+     ...
+   ```
+   
+1. After setup has completed, define the displayed environment variables. These variables provide the model training script with access credentials for your Watson Machine Learning service and Cloud Object Storage service. 
+
+   MacOS example:
+
+   ```
+   $ export ML_INSTANCE=...
```

Comment:
This needs to be brought in line with most recent training framework readme template

Reply:
Thanks Nick 🙂updated this!